java/src/main/java/org/rocksdb/Cache.java

```diff
@@ -10,4 +10,50 @@
   protected Cache(final long nativeHandle) {
     super(nativeHandle);
   }
+
+  /**
+   * @return the maximum configured capacity of the cache
+   * @throws RocksDBException
+   */
+  public long getCapacity() throws RocksDBException {
+    return getCapacity(nativeHandle_);
+  }
+
+  /**
+   * @return the memory size for the entries residing in the cache
+   * @throws RocksDBException
+   */
+  public long getUsage() throws RocksDBException {
+    return getUsage(nativeHandle_);
+  }
+
+  /**
+   * @return the memory size for the entries residing in the cache
+   * @throws RocksDBException
+   */
+  public long getHighPriorityPoolUsage() throws RocksDBException {
+    return getHighPriorityPoolUsage(nativeHandle_);
+  }
+
+  /**
+   * @return the memory size for the entries in use by the system
+   * @throws RocksDBException
+   */
+  public long getPinnedUsage() throws RocksDBException {
+    return getPinnedUsage(nativeHandle_);
+  }
+
+  /**
+   * @return the number of entries in the cache
+   * @throws RocksDBException
+   */
+  public long getEntries() throws RocksDBException {
+    return getEntries(nativeHandle_);
+  }
+
+  private native long getCapacity(final long handle) throws RocksDBException;
```

Comment:
It looks to me like these methods will never throw `RocksDBException`. If that is the case, perhaps remove the exceptions.